helm-charts/templates/data-plane/gateway-components/gateway-runtime/gateway-runtime-deployment.yaml

```diff
@@ -139,13 +221,18 @@ spec:
             defaultMode: 0777
         - name: router-keystore-secret-volume
           secret: 
+            {{- if and .Values.wso2.apk.dp.gatewayRuntime.deployment.router.configs .Values.wso2.apk.dp.gatewayRuntime.deployment.router.configs.tls }}
+            secretName: {{ .Values.wso2.apk.dp.gatewayRuntime.deployment.router.configs.secretName | default (printf "%s-router-server-cert" (include "apk-helm.resource.prefix" .)) }}
```

Comment:
Just to clarify, with the config: 
```
            configs:
              tls:
                secretName: "router-cert"
                privateKey: ""
                publicKey: ""
```
in values.yaml, won't this resolve secret name to `router-cert` 
and won't load the cert-manager cert which ends like `...-adapter-server-cert`